_delphi_utils_python/tests/test_covidcast_wrapper.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timedelta
+from pathlib import Path
+import pandas as pd
+from delphi_utils import covidcast_wrapper
+import covidcast
+from freezegun import freeze_time
+from delphi_epidata import Epidata
+from pandas.testing import assert_frame_equal
+
+TEST_DIR = Path(__file__).parent
+class TestCovidcastWrapper:
+    Epidata.debug = True
+    def test_metadata(self):
+        expected_df = covidcast.metadata()
+        df = covidcast_wrapper.metadata()
+        assert_frame_equal(expected_df, df)
+
+    @freeze_time("2022-01-29")
+    def test_signal(self):
+        meta_df = pd.read_pickle(f"{TEST_DIR}/test_data/covidcast_metadata.pkl")
```

Comment:
- What's the benefit of freeze_time here? It only affects calls like `datetime.today()` and I think you could just use `enddate = datetime(...)` instead. freeze_time makes me think that there's some internal calls using `today()` calls, but I don't think we rely on those anywhere?
- What's the benefit of pickling the metadata here? Since these tests are already making a lot of network calls, I'm leaning towards just "rounding" to making this script self-contained and not reliant on any pre-computed files. The benefit of making it self-contained is that if we ever need to test this wrapper later, we can just comment out the skip at the top, and run this test to compare the wrapper with the actual output of `metadata()` instead of a pre-computed one.

Reply:
moved as a script in testing_utils, hopefully that's good enough? also if this is something that's going to be one and done, I just used api calls everywhere and made the process quite expensive...should I make it bit less or trying to get the absolute truth?

Reply:
The script looks pretty good to me. I think we shouldn't worry about how expensive it is while testing and try to get a decent amount of data and compare it all. I'd rather spend an hour or two on a testing script run once than to spend hours deploying and reverting.